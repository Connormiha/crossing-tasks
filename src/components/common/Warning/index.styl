.warning
    background: #fff
    color: red
    position: absolute
    top: 10px
    right: 10px
    font-size: 20px
    padding: 3px 5px
