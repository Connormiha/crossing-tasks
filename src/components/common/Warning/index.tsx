const styles = require('./index.styl');

import * as React from 'react';

export default class Boat extends React.PureComponent<any, null> {
    render() {
        let {children} = this.props;

        return (
            <div className={styles.warning}>{children}</div>
        );
    }
}
