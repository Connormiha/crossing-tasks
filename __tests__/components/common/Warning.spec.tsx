import * as renderer from 'react-test-renderer';
import Warning from 'components/common/Warning';

describe('<Warning />', () => {
    it('should render', () => {
        let tree = renderer.create(
            <Warning>Foo</Warning>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
