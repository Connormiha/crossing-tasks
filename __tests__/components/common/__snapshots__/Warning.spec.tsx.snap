// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Warning /> should render 1`] = `
<div
  className="warning"
>
  Foo
</div>
`;
